zulipterminal/config/themes.py

```diff
@@ -49,6 +48,9 @@
     'area:msg': 'standout',
     'area:stream': 'standout',
     'area:error': 'standout',
+    'area:neutral': 'standout',
+    'task:success': 'standout',
+    'task:failure': 'standout',
```

Comment:
I am not sure if we currently have any use-case, but could we also support a `task:warning` perhaps? -for something that's not as severe as a failure.

Reply:
We could, though I'm unsure of what styles to expect for that. Could you suggest some color designs (which could potentially be rendered across all themes)?

Reply:
how about a `dark_yellow` or `light_orange`(not sure if they feasible, though). We could probably test other variants too.

Reply:
Well, we would tend to support all themes, so this might be a support issue for `gruvbox`. We could style it similar to `area:msg`, since its color scheme kind of aligns with what we call `warning` (?)

Reply:
Yep, sounds good!